### PR TITLE
add known issue about actor complexity

### PR DESCRIPTION
**Pull request by ryanatkn**

### CODE REVIEW

File: src/docs/known-issues.md [hunk 1]
```diff
@@ -82,3 +83,16 @@ Mitigations we've implemented:
   When you open our contextmenu on a link,
   you'll see the link again in the menu under your pointer by default,
   so to access your system's functionality on links, tap-and-hold twice.
+
+## Multi-actor systems are more complex
+
+Felt was designed from the beginning to support multiple users, or actors, for each account.
```
Comment (ryanatkn) [inline src/docs/known-issues.md:89]: Would be nice to make `Actors` a `<Vocab name="Actor">Actor</Vocab>` for the docs-integrated experience. Renaming these to Svelte and publishing them on the docs page is something to do during alpha.

File: src/docs/known-issues.md [hunk 2]
```diff
@@ -82,3 +83,16 @@ Mitigations we've implemented:
...
+This is a useful and powerful concept to integrate with a social app,
+and it helps support strong privacy mechanics with good UX.
+We're also excited to explore novel actor types with shared or automated control,
+like bot actors and group actors.
+
+This power brings complexity, and currently,
+end-users of felt-server are exposed to this concept and have no options for a simpler experience.
+Happily, our system is being designed to support a fully customizable UI,
```
Comment (greatbacon) [inline src/docs/known-issues.md:97]:
```suggestion
Fortunately, our system is being designed to support a fully customizable UI,
```

Comment (greatbacon) [reply src/docs/known-issues.md:97]: Doesn't have to be `Fortunately` but I think `Happily` reads weird here.

**Review by greatbacon — APPROVED** [review]

File: src/docs/known-issues.md (hunk 2 above)
Comment (ryanatkn) [reply src/docs/known-issues.md:97]: agree 💯

**Pull request merged**